packages/web-app/src/views/appeal-details/application-decision-date.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/two-thirds-page.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+
+{% block pageContent %}
+
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errorSummary
+    }) }}
+  {% endif %}
```

Comment:
The `govukErrorSummary` should already be included as part of the two-thirds-page layout